pkg/kubelet/kubeclientmanager/kubeclient_manager.go

```diff
@@ -77,28 +82,41 @@ func (manager *KubeClientManager) RegisterTenantSourceServer(source string, ref
 		return
 	}
 
-	if _, ok := manager.tenant2api[key]; !ok {
-		manager.tenant2api[key] = clientId
-		klog.Infof("added %d to the map, map has %+v", clientId, manager.tenant2api)
+	if _, ok := manager.puid2api[ref.UID]; !ok {
```

Comment:
Question: should we compare value if podUID is found in the map?

Reply:
will add warning log for this case.